src/Agent.Service/Windows/AgentService.cs

```diff
@@ -50,6 +50,7 @@ protected override void OnStart(string[] args)
                                 lock (ServiceLock)
                                 {
                                     AgentListener = CreateAgentListener();
+                                    WriteInfo(string.Format("Agent.Listner.exe arguments - {0}", AgentListener.StartInfo.Arguments));
```

Comment:
ok

---
In reply to: [119686929](https://github.com/Microsoft/vsts-agent/pull/985#discussion_r119686929) [](ancestors = 119686929)